client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -244,22 +243,14 @@ const ModelerSidebar = ({ modeler, robotId }) => {
       .saveXML({ format: true })
       .then((xml) => {
         parseBpmnToSsot(xml, robotId).then((result) => {
-          const ssot = JSON.stringify(result);
-          sessionStorage.setItem('ssotLocal', ssot);
+          if (typeof result === 'undefined') {
```

Comment:
Not sure, but I think my recommendation would be an easier solution to the defined state
```suggestion
          if (!!result) {
```

Reply:
ESlint then shows the warning `redundant double negotiation`. Thats why I didn't used this short form. And to be honest I think the typeof form of course is longer, but maybe better to understand.